src/Quartz/Impl/AdoJobStore/MySQLDelegate.cs

```diff
@@ -34,5 +34,10 @@ protected override string GetSelectNextTriggerToAcquireSql(int maxCount)
         {
             return SqlSelectNextTriggerToAcquire + " LIMIT " + maxCount;
         }
+
+        protected override string GetSelectNextMisfiredTriggersInStateToAcquireSql(int maxCount)
+        {
+            return SqlSelectHasMisfiredTriggersInState + " LIMIT " + maxCount;
```

Comment:
HasMisfiredTriggersInState has API promise "The most misfired triggers to return, negative for all", how does MySQL limit behave with -1?